github-orgs/kubeflow/org.yaml

```diff
@@ -677,6 +677,12 @@ orgs:
             allow_squash_merge: false
             description: Kubeflow SDK for ML Experience
             has_projects: true
+          docs-agent:
```

Comment:
can you keep this list alphabetical please ?

Reply:
done